chain/chain/src/chain.rs

```diff
@@ -913,7 +913,7 @@ impl Chain {
                     .validator_proposals
                     .iter()
                     .map(|validator_stake| {
-                        (validator_stake.amount / 1_000_000_000_000_000_000) as i64
+                        (validator_stake.stake / 1_000_000_000_000_000_000) as i64
```

Comment:
Hmm, the magic const is 10^18 instead of 10^24... Don't we have a named const somewhere?